### PR TITLE
Seperate question posting form template #910 - in progress

### PR DESCRIPTION
**Pull request by sandyvern**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [ ] all tests pass -- `rake test:all`
* [ ] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [ ] pull request is descriptively named with #number reference back to original issue
* [ ] if possible, multiple commits squashed if they're smaller changes
* [ ] reviewed/confirmed/tested by another contributor or maintainer
* [ ] `schema.rb.example` has been updated if any database migrations were added

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

**Comment by jywarren** [comment]
Hi, Sandy - looks like, since I merged in some new changes, you need to rebase your work on top of the latest master. If you remember, that'll be the process outlined here, but you may be able to skip step 2 if your master is still in sync with publiclab's master: https://publiclab.org/wiki/contributing-to-public-lab-software#Rewinding+the+master+branch

**Comment by sandyvern** [comment]
Hi Jeffery!
Ok...so just getting some time today to work on this. Because I stumbled over instructions getting setup with Cloud9...I decided to refork/clone and setup a new workspace on Cloud9. So have a few questions...
1) have the directions in the original issue changed since you merged some of the new changes? Should I still copy the form in post view to create a questions view? 
2)Should I create a new pull request since it is a new workspace? 
And lastly, I attempted to run tests with  rake test:all  but ran into errors saying I needed to run rake db:migrate ...do I need to run migrations (I didn't manage to figure out the testing situation in previous issues prior to submitting PRs just thought the testing went through TravisCI)? And I apologize for so many newbie question :-)


-----Original Message-----

**Comment by jywarren** [comment]
Taking a look at this shortly!

**Comment by jywarren** [comment]
No problem at all:

1) have the directions in the original issue changed since you merged some of the new changes? Should I still copy the form in post view to create a questions view?

I think the original issue still stands, but if there's a specific conflict I'm not noticing, please point it out -- sometimes it's hard to follow all the parallel changes :-)

2) Should I create a new pull request since it is a new workspace?

You should be able to `git push -f origin feature-branch-name` from any workspace, as it'll overwrite your PR commit.

> rake db:migrate

You know, I'd follow the instructions in the README as far as possible (so yes?), but if you're in an odd state and can't figure it out, you can share your Cloud9 instance with my Github username and I can have a look!

Thanks, Sandy!

**Comment by sandyvern** [comment]
I shared my Cloud9 instance with you so you can see where things are at currently cause I ended up opening a new pr last night with the c9 instance.

**Comment by jywarren** [comment]
Great, i'll take a look then. Can you close the PR you're not using? Thanks!

**Comment by jywarren** [comment]
Oh, hi @sandyvern - sorry, I think I was looking at this in the midst of the Barnraising event and lost track. I don't think I can find another PR, but I did find your workspace: https://github.com/publiclab/plots2/issues/928

Did you open a new PR, or could you? I can then help a bit with the rebase. 

Thanks, and sorry for the mixup!
